distribution/lib/Standard/Visualization/0.0.0-dev/src/Helpers.enso

```diff
@@ -247,6 +248,42 @@ Column.is_numeric : Boolean
 Column.is_numeric self =
     [Storage.Integer,Storage.Decimal].contains self.storage_type
 
+## PRIVATE
+   Returns the data requested to render a lazy view of the default visualisation. Decides
+   whether to return a simplified version of the lazy data format.
+make_lazy_visualisation_data : Text -> Vector Integer -> Vector Integer -> Text
+make_lazy_visualisation_data text text_window_position text_window_size chunk_size =
+    min_length_for_laziness = chunk_size * (text_window_size.first) * (text_window_size.second)
+    if text.length < min_length_for_laziness then text else
+        get_lazy_visualisation_text_window text text_window_position text_window_size chunk_size
+
+## UNSTABLE
+   ADVANCED
+   Returns the data requested to render a lazy view of the default visualisation.
+Any.to_lazy_visualization_data : Vector Integer -> Vector Integer -> Text
+Any.to_lazy_visualization_data self text_window_position text_window_size chunk_size =
+    ## Workaround so that the JS String is converted to a Text
+       https://www.pivotaltracker.com/story/show/184061302
+    "" + make_lazy_visualisation_data self.to_default_visualization_data text_window_position text_window_size chunk_size
+
+## UNSTABLE
+   ADVANCED
+   Returns the data requested to render a lazy view of the default visualisation.
+Text.to_default_visualization_data : Vector Integer -> Vector Integer -> Text
+Text.to_default_visualization_data self text_window_position=[0,0] text_window_size=[10,10] chunk_size=20 =
+    self.to_lazy_visualization_data text_window_position text_window_size chunk_size
+
+## UNSTABLE
+   ADVANCED
+   Returns the data requested to render a lazy view of the default visualisation.
+Text.to_default_visualization_data : Vector Integer -> Vector Integer -> Text
+Text.to_default_visualization_data self text_window_position=[0,0] text_window_size[10,10] chunk_size=20=
```

Comment:
Is the `to_default_visualization_data` method registered twice? Shouldn't it result into an error?